docs/start/framework/react/tutorial/reading-writing-file.md

```diff
@@ -50,12 +50,11 @@ pnpm i
 pnpm dev
 ```
 
-For this project, we'll need a few additional packages:
+For this project, we'll need the `uuid` package:
 
 ```bash
 # Install uuid for generating unique IDs
 pnpm add uuid
-pnpm add -D @types/uuid
 ```
```

Comment:
The `uuid` package now ships with its own type definitions, so `@types/uuid` is unnecessary